contrib/fossil/template.py

```diff
@@ -0,0 +1,20 @@
+pkgname = "fossil"
+pkgver = "2.23"
+pkgrel = 0
+build_style = "configure"
```

Comment:
anyhing preventing it from being gnu_configure?

Reply:
doesn't really matter, as long as it accepts the same configure arguments/works it should still be gnu_configure (sets up good defaults for cross-compiling, out of tree building and so on)

Reply:
> as long as it accepts the same configure arguments/works

Error: Unknown option --with-libtool-sysroot

Reply:
fyi to test cross `./cbuild -a aarch64 pkg contrib/fossil`

Reply:
it needs the same --host/--build etc as regular autotools to work (since it uses --build to invoke the native cc for a buildtime exe), but in this case that seems like Effort so just mark it as !cross